proto/umee/ugov/v1/genesis.proto

```diff
@@ -14,4 +16,9 @@ message GenesisState {
 
   // Emergency Group address
   string emergency_group = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // LiquidationParams is params for new liquidation inflation rate params  for staking denoms
+  LiquidationParams liquidation_params = 3 [(gogoproto.nullable) = false];
```

Comment:
Can we rename this?

Could be confused with Liquidation (the leverage module process)

Reply:
WDYT `InflationParams` instead of `LiquidationParams`
@robert-zaremba  @toteki